.github/workflows/ci.yml

```diff
@@ -1,10 +1,11 @@
+---
```

Comment:
Why this line?

Reply:
It is a document marker. YAML supports multiple documents per file. Anyway, my linter complains when it is not there. I shall turn that off. ;)

Reply:
I don't mind it being there, I was just curious :)